src/Common/tests/Performance/PerfUtils.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.IO;
+
+namespace System
+{
+    /// <summary>Static helper class for performance tests</summary>
+    public static class PerfUtils
+    {
+        /// <summary>
+        /// Helper method to create a string containing a number of random
+        /// characters equal to the specified length
+        /// </summary>
+        public static string CreateString(int length)
+        {
+            StringBuilder builder = new StringBuilder();
+            while (builder.Length < length)
+            {
+                string toAppend = Guid.NewGuid().ToString();
+                int toAppendLength = Math.Min(toAppend.Length, length - builder.Length);
+                builder.Append(toAppend, 0, toAppendLength);
+            }
+            return builder.ToString();
+        }
```

Comment:
For better or worse, this choice of string generation is going to potentially have an impact on the tests.  Using Guid.ToString means that the resulting strings will only ever have the characters 'a'-'f' and '0'-'9', which can play a role in the performance of things like compression.  Something to keep in mind.